packages/ia-components/sandbox/write-review/write-review.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import Button from './button/button';
+
+export default class WriteReview extends Component {
```

Comment:
Blocker: we will need to rework this main component and the feature expectations.